Input/Input.module.less

```diff
@@ -91,6 +96,23 @@
 			background-color: @agate-input-focused-background-color;
 			background-image: @agate-input-focused-background-image;
 			box-shadow: 0 0 0 @agate-input-decorator-focus-border-width @agate-input-decorator-focused-border-color;
+
+			.input {
+				&:focus {
+					opacity: @agate-input-focused-opacity;
+					color: @agate-input-focused-text-color;
+					caret-color: @agate-accent;
+					border-image-slice: 1;
+					border-image-source: @agate-input-active-border-image;
+					border-width: 0 0 @agate-input-focused-border-width;
+
+					&::placeholder {
+						opacity: @agate-input-opacity;
+					}
+				}
```

Comment:
Please add this effect (change underline color) when input component get focused as well. Currently, looks like spotlight lost even the spot is on the input component. Try with up/down/left/right key to change the focus on Gallium skin

Reply:
Thank you for the notice. I will update